css/main.css

```diff
@@ -1,3 +1,282 @@
-/* Add your CSS here */
+/* fonts
+font-family: 'Questrial', sans-serif;
+font-family: 'Raleway', sans-serif;
+font-family: 'Roboto', sans-serif;*/
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,400&display=swap');
```

Comment:
Do you need to import this css here? What would happen if you moved this font call to the other font calls in the html head?